.dockerignore

```diff
@@ -18,12 +18,9 @@
 build/*
 .gradle/*
 
+*.jar
```

Comment:
Why are we ignoring this?

Reply:
any *.jar that have been built locally should not get copied into the build container

Reply:
Ok, makes sense.